src/webui/src/components/public-child/OpenRow.tsx

```diff
@@ -105,7 +111,23 @@ class OpenRow extends React.Component<OpenRowProps, OpenRowState> {
                                         logCollection={EXPERIMENT.logCollectionEnabled}
                                     />
                                     :
-                                    <TrialLog logStr={logPathRow} id={trialId} />
+                                    <div>
+                                        <TrialLog logStr={logPathRow} id={trialId} />
+                                        {/* view each trial log in drawer*/}
+                                        <div id="trialog">
+                                            <div className="copy" style={{ marginTop: 15 }}>
+                                                <PrimaryButton
+                                                    onClick={this.openTrialLog.bind(this, 'TRIAL_LOG')}
```

Comment:
do you hide this button if the training service is not local?

Reply:
Yes, in https://github.com/microsoft/nni/blob/73989e06459120f9e954eb3edf55898ec4ecd2ae/src/webui/src/components/public-child/OpenRow.tsx#L106

Reply:
got it